### DISCUSSION TITLE
Renovate unable to parse dates for Schedules in config which should be valid according to the later js module

### DISCUSSION THREAD

**Discussion by jeremyhouser-ms** [renovatebot/renovate#29799] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps, Renovate version 37.351.1

### Please tell us more about your question or problem

Renovate is unable to parse the following content:

schedule: [
  "on 17th July 2024",
  "on 18th July 2024",
  "on 19th July 2024",
],

"topic": "Configuration Error",
"message": "Invalid packageRules[20].schedule: `Invalid schedule: Failed to parse \"on 17th July 2024\"`"

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Use the cron syntax if later doesn't work
